tests/lib/rook/base.py

```diff
@@ -70,16 +70,23 @@ def execute_in_ceph_toolbox(self, command, log_stdout=False):
     def install(self):
         # TODO(jhesketh): We may want to provide ways for tests to override
         #                 these
-        self.kubernetes.kubectl_apply(
-            os.path.join(self.ceph_dir, 'common.yaml'))
-        self.kubernetes.kubectl_apply(
-            os.path.join(self.ceph_dir, 'operator.yaml'))
+        self.kubernetes.kubectl("create namespace rook-ceph")
+        logger.info('Deploying rook operator with helm')
+        self.kubernetes.helm(f"install -n rook-ceph rook-ceph "
```

Comment:
Why is this in `rook/base.py`? Shouldn't this also be in `rook/ses.py`? It looks like because it differs both `ses.py` and `upstream.py` need to carry their own `install()`.

Reply:
Makes sense. I did not expect such deployment difference between upstream and ses.
I'll move code to ses class.

Reply:
After having a closer look at upstream install process with Helm is pretty much the same so I would keep it like that.

Reply:
Right, but we need to therefore update the upstream.py to pull in helm3 and then apply the appropriate chart.

Reply:
Helm support added for upstream